test/controllers/CreateOrAmendDividendsControllerSpec.scala

```diff
@@ -33,13 +33,13 @@ class CreateOrAmendDividendsControllerSpec extends TestUtils {
   val createOrAmendDividendsService: CreateOrAmendDividendsService = mock[CreateOrAmendDividendsService]
   val createOrAmendDividendsController = new CreateOrAmendDividendsController(createOrAmendDividendsService, mockControllerComponents, authorisedAction)
   val nino: String = "123456789"
-  val mtdItID: String = "123123123"
+  val mtditid: String = "123123123"
   val taxYear: Int = 1234
-  val badRequestModel = DesErrorBodyModel("INVALID_NINO", "Nino is invalid")
-  val notFoundModel = DesErrorBodyModel("NOT_FOUND_INCOME_SOURCE", "Can't find income source")
-  val serverErrorModel = DesErrorBodyModel("SERVER_ERROR", "Internal server error")
-  val serviceUnavailableErrorModel = DesErrorBodyModel("SERVICE_UNAVAILABLE", "Service is unavailable")
-  private val fakeGetRequest = FakeRequest("PUT", "/").withSession("MTDITID" -> "12234567890")
+  val badRequestModel: DesErrorBodyModel = DesErrorBodyModel("INVALID_NINO", "Nino is invalid")
+  val notFoundModel: DesErrorBodyModel = DesErrorBodyModel("NOT_FOUND_INCOME_SOURCE", "Can't find income source")
+  val serverErrorModel: DesErrorBodyModel = DesErrorBodyModel("SERVER_ERROR", "Internal server error")
+  val serviceUnavailableErrorModel: DesErrorBodyModel = DesErrorBodyModel("SERVICE_UNAVAILABLE", "Service is unavailable")
+  private val fakeGetRequest = FakeRequest("PUT", "/").withSession("MTDITID" -> "12234567890").withHeaders("mtditid" -> mtditid)
```

Comment:
Does it make sense to have an in session MTDITID that is unique from the one in the header?